src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -33,7 +33,7 @@
                 <TextBlock FontFamily="{x:Bind Name}"
                            Text="{x:Bind LocalizedName}" />
             </DataTemplate>
-
+            <model:IconPathConverter x:Key="IconSourceConverter" />

```

Comment:
looks unused? leftover from a previous commit maybe?

Reply:
Ah yes, thanks for the catch!